libs/ledger-live-common/src/wallet-api/useDappLogic.ts

```diff
@@ -386,7 +388,8 @@ export function useDappLogic({
             await new Promise<void>((resolve, reject) =>
               uiHook["account.request"]({
                 currencies: [getCryptoCurrencyById(requestedCurrency.currency)],
-                onSuccess: () => {
+                onSuccess: (account, parentAccount) => {
```

Comment:
`parentAccount` is not used causing an issue with linter on CI
```suggestion
                onSuccess: (account) => {
```